apps/aecore/test/aec_eunit_SUITE.erl

```diff
@@ -105,37 +104,37 @@ application_test(Config) ->
 
     app_stop(StartedApps -- ?TO_BE_STOPPED_APPS_BLACKLIST, TempDir).
 
-aec_sync_test(Config) ->
-    application:set_env(jobs, queues,
-                        [{sync_jobs, [passive]},
-                         {sync_workers, [{regulators, [{counter, [{limit, 4}]}]},
-                                         {producer, {aec_sync, sync_worker, []}}
-                                        ]}]),
-    {ok, StartedApps, TempDir} = prepare_app_start(aecore, Config),
-    ct:log("jobs running with env: ~p", [application:get_env(jobs, queues)]),
-    aec_test_utils:fake_start_aehttp(),
-
-    ok = application:start(aecore),
-    SyncPid = whereis(aec_sync),
-    ct:log("Running aec_sync ~p", [SyncPid]),
-
-    ok = aec_sync:connect_peer("http://my.evil_machine:3012/"),
-    ct:log("trying to connect --> job scheduled"),
-    %% give it some time to execute the scheduled job
-    timer:sleep(50),   
-    false = aec_peers:is_blocked("http://my.evil_machine:3012/"),
-    ct:log("as expected not blocked"),
-
-    ok = aec_sync:connect_peer("http://my.evil_machine:3012/"),
-    ct:log("trying to connect again --> job scheduled"),
-    timer:sleep(50),
-    SyncPid = whereis(aec_sync), %% Sync has not been restarted!
-    Peers = aec_peers:all(),
-    ct:log("tried twice and although it failed, it is a peer: ~p", [Peers]),
-    1 = length(Peers),
-
-    ok = application:stop(aecore),
-    app_stop(StartedApps -- ?TO_BE_STOPPED_APPS_BLACKLIST, TempDir).
+%% aec_sync_test(Config) ->
```

Comment:
Perhaps delete?

Reply:
I had some hope of recreating it... Let's keep it there as a reminder for me!?